examples/mocha/index.html

```diff
@@ -1,26 +1,21 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
It draws less attention to itself. It's no more important than any other element.